### PR TITLE
Sscs 10286 event state changes

### PR DESCRIPTION
**Pull request by dattipoeHMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by osheaf01 — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/sscs/jms/listener/HmcHearingsEventTopicListener.java [hunk 1]
```diff
@@ -52,8 +52,11 @@ public void onMessage(JmsBytesMessage message) throws JMSException, HmcEventProc
             HmcMessage hmcMessage = objectMapper.readValue(convertedMessage, HmcMessage.class);
 
             if (isMessageRelevantForService(hmcMessage)) {
-                log.info("Processing hearing ID: {} for case reference: {}", hmcMessage.getHearingId(),
-                    hmcMessage.getCaseId());
+                Long caseId = hmcMessage.getCaseId();
+                String hearingId = hmcMessage.getHearingId();
+
+                log.info("Attempting to process message from HMC hearings topic for Case ID {}, and Hearing ID {}.",
+                    caseId, hearingId);
```
Comment (nickwest1) [inline src/main/java/uk/gov/hmcts/reform/sscs/jms/listener/HmcHearingsEventTopicListener.java:59]: Minor: Not sure what value this adds over the previous way

File: src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/ProcessHmcMessageService.java [hunk 2]
```diff
@@ -26,64 +31,84 @@
...
+        if (eventMapper != null) {
+            EventType eventType = eventMapper.apply(hearingResponse);
+            if (eventType != null) {
+                ccdCaseService.updateCaseData(caseData,
+                    eventType,
+                    hmcStatus.getCcdUpdateSummary(),
+                    ccdUpdateDescription);
+            }
+        }
+    }
```
Comment (nickwest1) [inline src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/ProcessHmcMessageService.java:88]: Minor: Better to get out early.  I.e.
if(eventMapper == null ) {
return;
}
...

This way its easier to read with less nested if's

**Review by nickwest1 — APPROVED** [review]
A couple of comments for future PR's, but all good otherwise

**Pull request merged**